drivers/gpio/gpio_nct38xx_port.c

```diff
@@ -6,11 +6,10 @@
 
 #define DT_DRV_COMPAT nuvoton_nct38xx_gpio_port
 
-#include <zephyr/drivers/gpio.h>
```

Comment:
Whitespace changes (including header re-ordering) are best to be kept in a separate commit.